test/nftTrades.test.ts

```diff
@@ -108,6 +107,26 @@ describe('SwapRouter', () => {
       expect(methodParameters.calldata).to.eq(methodParametersV2.calldata)
       expect(methodParameters.value).to.eq(hexToDecimalString(methodParametersV2.value))
     })
+
+    it('encodes batch buying 2 ERC721s from LooksRare', async () => {
+      // buy items from block 17037140
+      const looksRareV2Data1: LooksRareV2Data = {
+        apiOrder: looksRareV2Orders[1],
+        taker: recipient,
+      }
+      const looksRareV2Data2: LooksRareV2Data = {
+        apiOrder: looksRareV2Orders[2],
+        taker: recipient,
```

Comment:
Can we get a test that has the zero address as the taker and ensure that the router sweeps the NFT back to the recipient after the trade?